packages/pluggableWidgets/web-view-native/src/WebView.tsx

```diff
@@ -44,7 +43,10 @@ export class WebView extends Component<Props> {
                     userAgent={this.props.userAgent}
                     onShouldStartLoadWithRequest={({ url }) => {
                         const openExternally =
-                            this.props.openLinksExternally && (html ? url.slice(0, 4) === "http" : url !== uri);
+                            this.props.openLinksExternally &&
+                            (html
+                                ? WebView.sanitize(url).startsWith("http")
```

Comment:
`WebView.sanitize(url)` is called twice. What about a variable here?

Reply:
Done